src/main.js

```diff
@@ -525,6 +526,44 @@ class MiniGraphCard extends LitElement {
       </svg>`;
   }
 
+  renderGridLines() {
+    const {
+      height, hours_to_show, grid_line_type,
```

Comment:
f we decide to move grid_line_type into show option - this line should be changed

Reply:
done